testing/unittests/dalitests.cpp

```diff
@@ -1731,9 +1731,9 @@ class CDaliDFSStressTests : public CppUnit::TestFixture
             ASSERT(strcmp(sfile3->querySubFile(0).queryLogicalName(), "regress::trans::sub3") == 0 && "promote failed, wrong name for sub3");
             ASSERT(outlinked.length() == 1 && "promote failed, outlinked expected only one item");
             ASSERT(strcmp(outlinked.popGet(), "regress::trans::sub2") == 0 && "promote failed, outlinked expected to be sub2");
-            Owned<IDistributedFile> sub1 = dir.lookup("regress::trans::sub1", user, false, false, false, NULL, timeout);
+            Owned<IDistributedFile> sub1 = dir.lookup("regress::trans::sub1", user, false, false, false, NULL, false, timeout);
             ASSERT(sub1.get() && "promote failed, sub1 was physically deleted");
-            Owned<IDistributedFile> sub2 = dir.lookup("regress::trans::sub2", user, false, false, false, NULL, timeout);
+            Owned<IDistributedFile> sub2 = dir.lookup("regress::trans::sub2", user, false, false, false, NULL, false, timeout);
             ASSERT(sub2.get() && "promote failed, sub2 was physically deleted");
```

Comment:
Can you add restricted access tests somewhere here (in dalitests.cpp) ?

Reply:
I was thinking of creating regression queries in testing/regress so that the handling of the restricted files by the different activities may be checked - both by signed and unsigned queries.  I'm not sure how useful calling lookup with PrivilegedUser=true and PrivilegedUser=false would be